src/Compilers/VisualBasic/Test/Emit/Attributes/InternalsVisibleToAndStrongNameTests.vb

```diff
@@ -472,7 +472,10 @@ End Class
         Assert.Equal(ERRID.ERR_PublicKeyContainerFailure, err.Code)
         Assert.Equal(2, err.Arguments.Count)
         Assert.Equal("foo", DirectCast(err.Arguments(0), String))
-        Assert.True(DirectCast(err.Arguments(1), String).Contains("HRESULT: 0x80090016"))
+        Dim errorText = DirectCast(err.Arguments(1), String)
```

Comment:
I'm not sure.  Maybe?

The issue is that the error text is coming from outside of Roslyn, so we can't the language-specific error message.

However, regardless of the language, the error contains the english text "HRESULT" and the string "0x80090016", which is what I'm testing here.  

If there's a better way to do this, then I'd really like to use that.